InternxtDesktop/Views/Backup/FolderSelectorView.swift

```diff
@@ -53,20 +47,36 @@ struct FolderSelectorView: View {
                         panel.canChooseFiles = false
                         if panel.runModal() == .OK {
                             for url in panel.urls {
-                                let foldername = url.lastPathComponent
-                                self.urls.append(url.absoluteString)
-                                self.foldernames.append(foldername)
+                                do {
+                                    if (!backupsService.urls.contains(url)) {
+                                        try self.backupsService.addFoldernameToBackup(
+                                          FoldernameToBackup(
+                                              url: url.absoluteString,
+                                              status: .selected
+                                          )
+                                        )
+                                    } else {
+                                        showErrorDialog(message: "Already added folder")
```

Comment:
drop the selected folder if a duplicated one is introduced, just ignore it, the user won't notice as the result is the same, but don't show an error dialog here